src/app/components/Toolbar/Features/Account/ManageableAccountDetails.tsx

```diff
@@ -16,14 +18,17 @@ import { AddressBox } from '../../../AddressBox'
 import { layerOverlayMinHeight } from '../layer'
 import { LayerContainer } from './../LayerContainer'
 import { uintToBase64, hex2uint } from '../../../../lib/helpers'
+import { DeleteAccount } from './DeleteAccount'
 
 interface ManageableAccountDetailsProps {
+  deleteWallet?: (address: string) => void
```

Comment:
Although we have inconsistent naming, i would use account here
```suggestion
  deleteAccount?: (address: string) => void
```

Reply:
And I would make it more explicit:
```suggestion
  /** If undefined: delete button is disabled */
  deleteAccount: undefined | (address: string) => void
```